lib/api/apiUtils/bucket/models/ReplicationConfiguration.js

```diff
@@ -315,6 +315,38 @@ class ReplicationConfiguration {
     }
 
     /**
+     * Get the XML representation of the configuration object
+     * @param {object} config - The bucket replication configuration
+     * @return {string} - The XML representation of the configuration
+     */
+    static getConfigXML(config) {
+        const { role, destination, rules } = config;
+        const Role = `<Role>${role}</Role>`;
+        const Bucket = `<Bucket>${destination}</Bucket>`;
+        const rulesXML = rules.map(rule => {
+            const { prefix, enabled, storageClass, id } = rule;
+            const Prefix = prefix === '' ? '<Prefix/>' :
+                `<Prefix>${prefix}</Prefix>`;
+            const Status =
+                `<Status>${enabled ? 'Enabled' : 'Disabled'}</Status>`;
+            const StorageClass = storageClass ?
+                `<StorageClass>${storageClass}</StorageClass>` : '';
+            const Destination =
+                `<Destination>${Bucket + StorageClass}</Destination>`;
```

Comment:
minor: a more straghtforward and consistent way to concatenate here would be `${Bucket}${StorageClass}`